### PR TITLE
Fix an instance that used archivesize where it ought to use size

### PR DESCRIPTION
**Pull request by dralley**
Presumably this code is meaning to use the installed size, not the size including CPIO headers.

### CODE REVIEW

File: lib/psm.c [hunk 1]
```diff
@@ -104,7 +104,7 @@ static rpmRC markReplacedFiles(const rpmpsm psm)
 
 	if (!headerGet(h, RPMTAG_FILESTATES, &secStates, HEADERGET_MINMEM))
 	    continue;
-	
+
```
Comment (dralley) [inline lib/psm.c:107]: Ignore the editor autoformat noise.

**Comment by pmatilai** [comment]
You already suggested this in some ticket and I think I already said no: on install, the progress is measured from the archive position, not the installed size. Could it be different, sure, but there's also no need to change it just for change's sake.

File: lib/psm.c (hunk 1 above)
Comment (pmatilai) [reply lib/psm.c:107]: For submitting patches, you need to get rid of any editor caused artifacts. Patches with random whitespace changes will not get merged.

**Comment by dralley** [comment]
Sorry if so, I don't recall seeing a response on that specific topic.

**Pull request closed without merging**